Index.html

```diff
@@ -129,13 +129,13 @@ <h2>About me</h2>
            </div> 
         </div>
 
-        <div class="form">
+        <div class="form-section">
             <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-            <form class="form-entries">
-                <input type="text" class="fill-area" name="fname" placeholder="Full Name">
-                <input type="text" class="fill-area" name="email" placeholder="Email Address">
-                <textarea rows="8" placeholder="Enter text here"></textarea>
-                <input class="button-1" id="submit" type="submit" value="Get in touch">
+            <form action="https://formspree.io/f/mrgdnqrz" method="post">
+                <input type="text" class="fill-area" name="full-name" required maxlength="30" placeholder="Full Name">
+                <input type="email" class="fill-area" name="email" required placeholder="Email Address">
```

Comment:
- [ ] Please kindly increase the padding of these two element in order to have a design that is close to that of figma. 👍 

## Figma Design
![eeeeetttt](https://user-images.githubusercontent.com/103386183/205922089-112d4387-6853-4a92-9d6d-2e991790c9dd.png)

Reply:
Thank you, Priscilla. I have made the change to the field padding, and it should closely mimic the figma design now.